comictaggerlib/ui/taggerwindow.ui

```diff
@@ -59,30 +59,13 @@
             <property name="formAlignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
-            <item row="1" column="0">
-             <widget class="QLabel" name="label">
-              <property name="text">
-               <string>Read Style</string>
-              </property>
-             </widget>
-            </item>
-            <item row="1" column="1">
-             <widget class="QComboBox" name="cbLoadDataStyle"/>
-            </item>
-            <item row="2" column="0">
-             <widget class="QLabel" name="saveStyleLabel">
-              <property name="text">
-               <string>Modify Style</string>
-              </property>
-             </widget>
-            </item>
-            <item row="2" column="1">
-             <widget class="QComboBox" name="cbSaveDataStyle"/>
-            </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_md_source">
               <property name="text">
-               <string>Metadata Source</string>
+               <string>Data Source</string>
```

Comment:
Why was this renamed?

Reply:
I renamed it because it takes up too much space and means the more important names of the source or styles get more truncated.